src/hexkit/providers/s3/provider.py

```diff
@@ -208,35 +204,29 @@ def _translate_s3_client_errors(  # noqa: C901
             exception = cls.ObjectAlreadyExistsError(
                 bucket_id=bucket_id, object_id=object_id
             )
-        elif error_code == "ObjectAlreadyInActiveTierError":
```

Comment:
Why did this get deleted? Are we not using it anymore?

Reply:
This branch was exactly duplicated directly above

Reply:
Okay, it didn't show in the diff (guess I could have viewed the file)

Reply:
Would not have seen that either, if ruff did not complain